thinkhazard/static/less/icons.less

```diff
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: 'thinkhazard';
+  src: url('../webfonts/thinkhazard/fonts/thinkhazard.eot?-xr90r9');
+  src: url('../webfonts/thinkhazard/fonts/thinkhazard.eot?#iefix-xr90r9') format('embedded-opentype'),
+       url('../webfonts/thinkhazard/fonts/thinkhazard.ttf?-xr90r9') format('truetype'),
+       url('../webfonts/thinkhazard/fonts/thinkhazard.woff?-xr90r9') format('woff'),
+       url('../webfonts/thinkhazard/fonts/thinkhazard.svg?-xr90r9#thinkhazard') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+
+[class^="icon-"], [class*=" icon-"] {
```

Comment:
```
[class^="foo"]
```
The above means that any element with a class prefixed with `foo` will match.

```
[class*=" foo"]
```
The above means that any element with a class containing ` foo` (note the space) will match.

https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors